modules/nw-sriov-hwol-using-vf-management-port.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-hardware-offloading.adoc
+
+:_content-type: PROCEDURE
+[id="using-vf-improve-network-traffic-performance_{context}"]
+= Using a virtual function to improve network traffic performance
+
+Follow this procedure to assign a virtual function to the OVN-Kubernetes management port and increase its network traffic performance.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Add the `network.operator.openshift.io/smart-nic` label to each worker node with a SmartNIC present by running the following command:
++
+[source,terminal]
+----
+$ oc label node <node-name> network.operator.openshift.io/smart-nic=
+----
++
+Use the `oc get nodes` command to get a list of the available nodes.
+
+. Create a policy named `sriov-node-policy.yaml` with content such as the following example:
```

Comment:
We need a specific explanation of the pfNames portion. Typically in OCP docs, you have a number (1) in the yaml and an explanation below that highlights that the user needs to change that bit. It would be more clear to explain that `ens8f0` needs to be replaced by the netdev of their choice. Also, I would elaborate on the intent at the beginning of the doc:


Use the following steps to assign a virtual function to the OVN-Kubernetes management port and enhance its network traffic performance. The objective is to establish two pools: one pool containing a single virtual function, which will be utilized by ovn-k, and a second pool comprising the remaining virtual functions (in this case, five as illustrated below).

Reply:
Great points, thank you!

Reply:
Good points Balazs. I would additionally add that fields such as ```numVfs: 6``` are provided here as an example. Users should have ```numVfs``` be set to reasonable value (we can refer to the other section which talks about the sriov policies).

Reply:
Are there any other fields that I should additional information to, or just `numVfs`?

Reply:
@wizhaoredhat I'm also not familiar with the docs in this area in general. What other section are you referring to which talks about SR-IOV policies? Is there a specific area that should be linked to?

Reply:
For example this section here: https://docs.openshift.com/container-platform/4.11/networking/hardware_networks/configuring-sriov-device.html#nw-sriov-networknodepolicy-object_configuring-sriov-device

Reply:
Updated to include the linked section in the "Additional resources" for this module. The `numVfs` section of each YAML file is annotated with a description that guides the user to the "SR-IOV network node configuration object" section for more information.